docs/public/s2i-to-devfile.adoc

```diff
@@ -0,0 +1,90 @@
+== Migrating from S2I (Source-to-Image) components to Devfile components
+
+https://devfile.github.io/[Devfiles] are new way of deploying component with odo.
+
+Existing users using S2I based components can migrate to a devfile component by following the below steps:
+
+. Change to the component directory you want to migrate:
++
+[source,sh]
+----
+ $ cd <directory-name>
+----
+
+. List the component status 
++
+[source,sh]
+----
+ $ odo list
+
+Openshift Components: 
+APP     NAME      PROJECT        TYPE       SOURCETYPE     STATE
+app     myapp     myproject      nodejs     local          Pushed
+
+----
+
+. Convert s2i component to devfile component by running the following command:
```

Comment:
S2I needs to be capitalized

Reply:
`Convert the S2I component to a devfile component by running the following command:`